controllers/testdata/deploy/case_1.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: dspipelines.opendatahub.io/v1alpha1
+kind: DSPipeline
+metadata:
+  name: testdsp
+spec:
+  apiServer:
+    artifactScriptConfigMap:
+      name: ds-pipeline-artifact-script-testdsp
+      key: "artifact_script"
+    apiServerImage: quay.io/modh/odh-ml-pipelines-api-server-container:v1.18.0-8
```

Comment:
@rimolive @gmfrasca do we have upstream images we can use?

Reply:
out of scope for this pr, should be a github issue

Reply:
#10